modules/validations/empty_start_date.js

```diff
@@ -0,0 +1,49 @@
+import { t } from '../core/localizer';
+import { utilDisplayLabel } from '../util';
+import { validationIssue, validationIssueFix } from '../core/validation';
+
+export function validationEmptyStartDate() {
+  const type = 'empty_start_date';
+
+  const validation = function checkEmptyStartDate(entity) {
+    // If start_date is not empty, return nothing
+    if (entity.tags && (entity.tags.start_date || entity.tags['start_date:edtf'])) return [];
+    // If entity has no tags, return nothing
+    if (Object.keys(entity.tags).length === 0) return [];
+    // Rule should be ignored for natural entities and waterways
+    if (entity.tags && (entity.tags.natural || entity.tags.waterway || entity.tags.water)) return [];
```

Comment:
These three keys represent the majority of “timeless” features, but we might still want to factor the logic out into a separate array or something. (Maybe eventually when we start using [our id-tagging-schema fork](https://github.com/OpenHistoricalMap/id-tagging-schema/), we’ll have it indicate which keys can make a feature timeless, to make this rule more data-driven.)

Reply:
I created `osmTimelessFeatureTagValues` to check for that. It uses the tag values that I found while using the ui, if there are any other ones they would have to be added. Is there any documentation on all the possible values for the tags? It would help check that.